### ISSUE TITLE
mt5-base model (unicamp-dl/mt5-base-en-msmarco) throws 'nan' outputs

### ISSUE THREAD

**Issue by cramraj8** [beir-cellar/beir#173]
Hi,

I was running `unicamp-dl/mt5-base-en-msmarco`: ['▁no' , '▁yes'] model for both English and other My.TyDi languages, but the output scores are **nan**. When I switched to `unicamp-dl/mt5-13b-mmarco-100k`: ['▁', '▁true'] model, I get actual real logits. I wonder if there's any issues with underlying `unicamp-dl/mt5-base-en-msmarco` model.

Thanks.

**Comment by thakur-nandan** [comment]
I sadly am not aware of the model and what the issue might be here.
